includes/RequestParser.hpp

```diff
@@ -1,26 +1,56 @@
-#ifndef WEBSERV_REQUESTFILEPARSER_HPP
-#define WEBSERV_REQUESTFILEPARSER_HPP
+//
```

Comment:
Next Pull Request I do I will delete all those headers... it was Clion that automatically created them and we don't need them or to specify who created the file (we can see who created what on the IDE with Git Blame or something like this)

Reply:
All right,
Also maybe the colors can maybe be moved somewhere to be accessible in all files, if we will be using them